### DISCUSSION TITLE
How can I find all people which are dead but without a specific death date?

### DISCUSSION THREAD

**Discussion by Tom-Furrer** [fisharebest/webtrees#4914] [Q&A]
There is a checkbox option to add death as
_"This event occurred, but the details are unknown."_

How can I create a list of people missing a specific death date but have set the "this event occured..." checkbox?

**Comment by fisharebest** [comment]
There is a report "missing data".

Tip: for questions about using webtrees, you will often get a much faster answer on the forum at www.webtrees.net

**Reply by Tom-Furrer** [reply]
Thanks for your hint about "missing data" ("Fehlende Daten").

But how can I produce a list with the above criterias?
(I don't see how I could choose "event occured, but details are unknown").

Could you please tell me how to produce such a list?

**Comment by fisharebest** [comment]
Oops - just realised you posted this in the "discussions", rather than the "issues".  (Nobody has really used the discuss feature here, and I forgot it existed!)

**Comment by fisharebest** [comment]
The report will list individuals where the event occured, and either the date/place is unknown.  Just select the type of event (birth, death, etc.) that you are interested in.

**Reply by Tom-Furrer** [reply]
I'm not sure if I understand this report:
If I choose myself and ALL ALL and check only "Death" I get a list including myself (I'm neather death nor I have the event set).
Also my wife, my parents and many others - all still living - appear on this list.

I'm looking only for those individuals that have 
`1 DEAT Y`
in the *.ged file. 
There should be currently only 5 people on this report (after checking the exported *.ged file)
